src/js/directives/index.js

```diff
@@ -2,7 +2,13 @@ import {app} from '../bootstrap';
 import HeaderDirective from './HeaderDirective';
 import FooterDirective from './FooterDirective';
 import UnauthorizedDirective from './UnauthorizedDirective';
+import {preview as RecipePreviewDirective, search as RecipeSearchDirective} from './RecipeDirective';
+import {view as RecipeViewDirective, liked as RecipeLikedDirective} from './RecipeDirective';
 
 app.directive('header', HeaderDirective);
 app.directive('footer', FooterDirective);
 app.directive('unauthorized', UnauthorizedDirective);
+app.directive('recipepreview', RecipePreviewDirective);
+app.directive('recipesearch', RecipeSearchDirective);
+app.directive('recipeview', RecipeViewDirective);
+app.directive('recipeliked', RecipeLikedDirective);
```

Comment:
when I do the call for the directive in HTML it can be done with `RecipeLiked` or `recipeLiked` I'm pretty sure. At least the latter works.